### PR TITLE
Basic Layout Created

### PR DESCRIPTION
**Pull request by T0ny-X**
image
The Text " Magic Kitchen" is a placeholder where any header or content would be.

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Heading } from '@chakra-ui/react';
+import Navbar from "./Navbar";
+
+const Header = () => {
+  return (
+    <header className="App-header">
+        <Navbar />
+        <Heading as="h1" size="lg" fontWeight="bold">
+          Magic Kitchen
```
Comment (hetd54) [inline src/components/Header.jsx:10]: Ask Anna if she wants the full logo here or if she wants the page name?

File: src/App.js
Comment (hetd54) [inline src/App.js:1]: We do not need App.js, you can delete this file. We will import your 'Layout.js' file within the routes folder pages.

**Review by hetd54 — CHANGES_REQUESTED** [review]

File: src/components/Header.jsx (hunk 1 above)
Comment (T0ny-X) [reply src/components/Header.jsx:10]: @ah0606

**Review by hetd54 — APPROVED** [review]
🚀

File: src/components/Layout.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const Layout = ({ children }) => {
+  // Add prop validation with PropTypes
+  Layout.propTypes = {
+    children: PropTypes.node.isRequired, // Require children prop
+  };
+
+  return (
+    <>
+      <Header />
+      {children}
```
Comment (hetd54) [inline src/components/Layout.js:15]: Can we wrap children in a Box component and make it take up the full view of the page by default? We want Footer to be at the bottom of the screen.

See (https://chakra-ui.com/docs/components/box) -- probably want the height value to be 100vh

**Review by hetd54 — CHANGES_REQUESTED** [review]
Sorry Tony, just noticed a styling thing

File: src/components/Layout.js (hunk 2 above)
Comment (hetd54) [reply src/components/Layout.js:15]: Or a container actually: https://chakra-ui.com/docs/components/container

Look @ the example near the bottom @T0ny-X !

**Review by hetd54 — APPROVED** [review]
Thanks Tony!

**Review by hjones123 — APPROVED** [review]
Looking good!

**Pull request merged**